app/views/notifications/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= simple_form_for(@notification) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.association :event %>
+    <%= f.input :status, as: :select, collection: Notification::VALID_STATUSES %>
+    <%= f.input :hours_to_try %>
+    <%= f.input :minutes_interval %>
+    <%= f.input :attempts_before_escalation %>
+    <%= f.input :start_at %>
+    <%= f.input :started_at %>
```

Comment:
This should not be on the form, it is filled in by the system.
It would be nice, but not required, if you could add some hints and/or labels to the inputs.